test/probes/http-common.js

```diff
@@ -917,10 +915,13 @@ describe(`probes.${p}`, function () {
           emitter,
           function (done) {
             const req = driver.get(url, function (res) {
+              // when we get data abort the request
```

Comment:
I think for future readers of the code it would interesting to know **why** the request is aborted, resp. why there is a delay needed on line 897.
The comments explain what is being done--which can be read from the code--but the why is less obvious.

Reply:
fair comment. this is what i'm working on now and will expand the comment.

Reply:
but i think the why is stated in the test - make sure that an error is not reported when the client aborts a connection. make sense?